c10/core/DispatchKeySet.h

```diff
@@ -203,6 +203,9 @@ constexpr DispatchKeySet autograd_dispatch_keyset = DispatchKeySet({
     DispatchKey::AutogradOther,
 });
 
+constexpr DispatchKeySet autograd_inplace_dispatch_keyset =
```

Comment:
My understanding for the splitted AutogradCPU, AutogradCUDA, etc keys was that each backend can register a custom VariableType kernel if they want to.
So I guess that some backends are using that feature?

And it's these backends I mention above.

Reply:
Oh right, for AutogradXLA it's not a big problem since we only customize 2 operators using it for now. It might be for AutogradMLC tho, depends on how they want to support view/inplace...